powersimdata/scenario/create.py

```diff
@@ -356,20 +336,19 @@ def get_base_profile(self, kind):
         :param str kind: one of *'demand'*, *'hydro'*, *'solar'*, *'wind'*.
         :return: (*list*) -- available version for selected profile kind.
         """
-        return self.profile.get_base_profile(kind)
+        id = InputData()
```

Comment:
I'd convert this to one line, partly to remove `id` as a variable name, since it's also a keyword

Reply:
Done